mon-pix/app/styles/pages/_challenge.scss

```
@@ -120,3 +110,23 @@
     letter-spacing: 0.009rem;
   }
 }
+
+.focus-zone-warning--triggered{
+  z-index: 1;
+  background-color: $pix-neutral-20;
+  border-radius: 5px;
+  outline: 5px dashed $pix-shades-100;
+  outline-offset: -5px;
+}
+
+.focus-zone-warning--overlay {
```

```suggestion
.focus-zone-warning__overlay {
```

**refactor:** convention BEM, c'est un élément et pas un modificateur.